lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -291,6 +294,12 @@ def add_test_targets
               create_test_target_embed_frameworks_script(test_type)
               create_test_target_copy_resources_script(test_type)
 
+              # Generate vanila Info.plist for test target similar to the one xcode gererates for new test target.
+              # This creates valid test bundle accessible at the runtime, allowing tests to load bundle resources
+              # defined in podspec.
+              test_info_plist_path = target.support_files_dir + "#{test_type.capitalize}TestInfo.plist"
```

Comment:
very nit but move this into a method in `pod_target.rb`

```ruby
    # @param  [Symbol] test_type
    #         The test type this Info.plist path is for.
    #
    # @return [Pathname] The absolute path of the Info.plist for the given test type.
    #    
    def info_plist_path_for_test_type(test_type)
      support_files_dir + "#{test_target_label(test_type)}-Info.plist"
    end
```

Then call it from here. We will soon be refactoring `pod_target.rb` but for now it's OK to add this there.

Reply:
yes, looks like opportunity to refactor, will do